Source/Initialization/WarpXInitData.cpp

```diff
@@ -392,10 +392,6 @@ WarpX::InitData ()
         WarpX::InitNCICorrector();
     }
 
-    if (WarpX::use_filter) {
```

Comment:
Yes, I think this is because otherwise these lines added in WarpX.cpp
```cpp
    if (use_filter)
    {
        ngJ += bilinear_filter.stencil_length_each_dir - amrex::IntVect(1);
    }
```
would be ineffective: `bilinear_filter.stencil_length_each_dir` would not be initialized at this point, if `InitFilter` were called from within `InitData` as before.